### DISCUSSION TITLE
card reader

### DISCUSSION THREAD

**Discussion by robi62** [Leo3213/HP-ProBook-450-G3-hackintosh-bigsur-11.3#15] [General]
Hi card reader works If you want kext I use let me know

**Comment by Leo3213** [comment]
Will do I'll try it out

Sorry I'm college and I've been swamped 😂

**Comment by robi62** [comment]
I did you just ask me which kexts I use????

**Reply by Leo3213** [reply]
Originally yes because i was getting the kext you shared confused with another lol
